### PR TITLE
fix(upgrade): add DB URL to log message at end up upgrade

### LINKED ISSUE #20255: [2.0] Upgrade Logging -- specify where to login to the UI
For the final, successful log message in the `upgrade` command from v1 to v2....it currently reads:
`
{"level":"info","ts":1607098245.4032829,"caller":"upgrade/upgrade.go:457","msg":"Upgrade successfully completed. Start service now"}
`

Request is to add the URL where to access the UI from.  Something like:
`
{"level":"info","ts":1607098245.4032829,"caller":"upgrade/upgrade.go:457","msg":"Upgrade successfully completed. Start service now then log in via http://localhost:8086"}
`

### PR DESCRIPTION
**Pull request by danxmoran**
Closes #20255

### CODE REVIEW

File: cmd/influxd/upgrade/upgrade.go [hunk 1]
```diff
@@ -453,7 +455,10 @@ func runUpgradeE(*cobra.Command, []string) error {
 		)
 	}
 
-	log.Info("Upgrade successfully completed. Start service now")
+	log.Info(
+		"Upgrade successfully completed. Start service now to log in",
+		zap.String("login_url", options.source.dbURL),
```
Comment (danxmoran) [inline cmd/influxd/upgrade/upgrade.go:460]: @timhallinflux this ends up logging lines like:
```
{"level":"info","ts":1607373509.091895,"caller":"upgrade/upgrade.go:458","msg":"Upgrade successfully completed. Start service now to log in","login_url":"http://localhost:8086"}
```
Look good to you?

Comment (timhallinflux) [reply cmd/influxd/upgrade/upgrade.go:460]: +1

Comment (lesam) [reply cmd/influxd/upgrade/upgrade.go:460]: "Start service now to login" sounds to me like going to http://localhost:8086 will start the service for me - for a user that just ran the upgrade this is not the case.

Maybe `Upgrade successfully completed. Ensure influxd service is started and then log in","login_url":"http://localhost:8086"}` or similar would be clearer?

Comment (danxmoran) [reply cmd/influxd/upgrade/upgrade.go:460]: "Ensure influxd service is started" sounds to me like we expect `upgrade` to start the service as its last step. How does the latest push look?

**Review by lesam — APPROVED** [review]
lgtm

**Pull request merged**